frontend/src/views/CanteensPage/CanteensHome.vue

```diff
@@ -230,6 +229,15 @@ export default {
     validators() {
       return validators
     },
+    resultsCountText() {
+      if (this.hasActiveFilter || this.searchTerm) {
```

Comment:
Le texte change avec chaque `keydown` de `searchTerm` même si les resultats ne sont pas encore affichés - est-ce que c'est prévu ?

Reply:
Bien vu, c'est plutôt le `this.$route.query.recherche` qui doit être pris en compte